source/backups.rst

```diff
@@ -138,6 +138,40 @@ When the backup destination is configured and applied with `kubectl apply -f dep
         storageName: s3-us-west
       EOF
 
+.. _backups-pitr-binlog:
+
+Backing up binary logs for point-in-time-recovery
+--------------------------------------------------
+
+Point-in-time-recovery functionality allows users to roll back the cluster to a
+specific transaction, time (or even skip a transaction in some cases).
+Technically, this feature involves continuously saving binary log updates to the
+backup storage. Point-in-time-recovery is off by default.
```

Comment:
I think we should mention that it would be good when enabling PITR that the bucket/directory which holds the binlogs should be empty (without some binlogs/files from previous attempts or other clusters) - otherwise you might end up with some issues.